apps/meeting/serverless/src/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "chime-sdk-meeting-serverless-demos",
+  "version": "0.1.0",
+  "description": "Amazon Chime SDK Serverless Demos",
+  "dependencies": {
+    "aws-sdk": "^2.1034.0"
+  },
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/aws-samples/amazon-chime-sdk"
+  }
+}
```

Comment:
Have you tried running the demo without this `package.json` and see if it works? Since the settings is the same in the meeting `package.json` file

Reply:
yes it works. not sure why it didn't work in the begining. I removed the related code.